autovizwidget/requirements.txt

```diff
@@ -1,5 +1,5 @@
 plotly>=3
 ipywidgets>5.0.0
-hdijupyterutils>=0.6
+git+https://github.com/viaduct-ai/sparkmagic.git@4242ac83059bfa736c785d353d1ac66bc424a501#egg=hdijupyterutils&subdirectory=hdijupyterutils
```

Comment:
Is this needed for the tests? Is it possible to use a relative directory?

Reply:
lemme give it a try - would certainly make my life easier